randovania/games/samus_returns/layout/msr_cosmetic_patches.py

```diff
@@ -48,6 +48,7 @@ class MSRCosmeticPatches(BaseCosmeticPatches):
     aeion_bar_color: tuple[int, int, int] = DEFAULT_AEION_BAR_COLOR
     ammo_hud_color: tuple[int, int, int] = DEFAULT_AMMO_HUD_COLOR
     show_room_names: MSRRoomGuiType = MSRRoomGuiType.NONE
+    enable_remote_lua: bool = False
```

Comment:
AFAIK this needs a migration

Reply:
I checked it with a json file from version 32 to 33 and it just added the default of false with the next start.

Reply:
huh, ok then.

Reply:
The reason of a migration is to not break with older Randovania versions. If after using the new version, particularly tweaking this setting, you open current stable it'll either complain about an unknown field or transparently revert this field.

Reply:
Wait, what? How can it transparently revert a field? An older version can not know about a future field. It could only open the highest schema version it knows from the user data folder.
But afaik, if we add a new field with a default value we just bump the schema version, which is already done because of the `input_exheader` migration.

Reply:
That's what I meant, a migration is needed always, it can just be empty. Haven't looked at the code for the PR 😅